tests/test_pdb.py

```diff
@@ -312,11 +312,16 @@ def test_multichain_load_cycle(get_fn):
     assert len(bonds) == len(bonds2)
 
 def test_load_pdb_input_top(get_fn):
-
     pdb = get_fn('native.pdb')
     p_1 = load_pdb(pdb)
 
     p_2 = load_pdb(pdb, top=p_1.topology)
 
     eq(p_1.xyz, p_2.xyz)
     eq(p_1.topology, p_2.topology)
+
+def test_chimera_indexing(get_fn):
+    load_pdb(get_fn('chimera_indexing.pdb'))  # this should just not fail
+
+def test_vmd_indexing(get_fn):
+    load_pdb(get_fn('vmd_indexing.pdb'))  # this should just not fail
```

Comment:
Could we add some (superficial) checks that i.e. the atom numbers and positions are what's expected?

Reply:
Done!